consensus/bor/bor.go

```diff
@@ -172,8 +172,13 @@ func encodeSigHeader(w io.Writer, header *types.Header) {
 		header.Extra[:len(header.Extra)-65], // Yes, this will panic if extra is too short
 		header.MixDigest,
 		header.Nonce,
-	})
-	if err != nil {
+	}
+	if c.JaipurBlock >= int(header.Number.Int64()) {
```

Comment:
Shouldn't we use big.Int to compare?

Reply:
It does not matter here, Number in header is a big.int just for convenience but the block number will never be higher than int64.